dockstore-common/src/main/scala/io/dockstore/common/WdlBridge.scala

```diff
@@ -136,7 +136,7 @@ class WdlBridge {
 
     val workflowName = primaryCallable.name
     primaryCallable.inputs
-      .filter(input => input.womType.stableName.toString.equals("File") || input.womType.stableName.toString.equals("Array[File]"))
+      .filter(input => input.womType.stableName.toString.contains("File"))
```

Comment:
Should be able to narrow this down. 
Something like `equals("File") || equals ("File?") || equals ("File[]") || something else`? or `startsWith("File")`

or `replaceAll("?$", "").equals(whatever)`

I want to avoid this picking up a something like `Not-A-File` down the road